### ISSUE TITLE
fix(platform): icon tab bar a11y issues with spacing

### ISSUE THREAD

**Issue by dmitry-stepanenko** [SAP/fundamental-ngx#7031]
#### Is this a bug, enhancement, or feature request?
There're couple of issues with a11y related to spacing:
**Issue 1:**
Observed Behavior: On text spacing the screen, the below tabs are truncated and not visible to the user properly. 

Expected Behavior: Ideally, on text spacing any UI shouldn’t get truncated and UI should be visible to the user.
![image](https://user-images.githubusercontent.com/33101123/139065394-ee997fd1-3f86-4a24-929d-48a874f7114f.png)

**Issue 2:**
 Observed Behavior: On text spacing the ‘More’ button is overlapped on the tab. 

Expected Behavior: ideally, on different view of the screen the UI should be properly visible to the user.
![image](https://user-images.githubusercontent.com/33101123/139065580-17241b16-3e9b-44f5-a8fc-794e27b410cc.png)

These issues were originally reported on https://github.com/SAP/fundamental-ngx/issues/6752 as part of the document:
[Icon Tab Bar.docx](https://github.com/SAP/fundamental-ngx/files/7241882/Icon.Tab.Bar.docx)
#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
@fundamental-ngx@0.33
